python/python/lancedb/table.py

```diff
@@ -500,6 +501,12 @@ def create_fts_index(
         use_tantivy: bool, default True
             If True, use the legacy full-text search implementation based on tantivy.
             If False, use the new full-text search implementation based on lance-index.
+        with_position: bool, default True
+            Only available with use_tantivy=False
+            If False, do not store the positions of the terms in the text.
+            This can reduce the size of the index and improve indexing speed.
+            But it will not be possible to use phrase queries.
```

Comment:
the current behavior is a little confusing, if not all words are known, then it would return empty result, otherwise it raise an exception to inform the users to create the index with position

Reply:
going to fix it